app/Jobs/ImportFacebookSharePosts.php

```diff
@@ -1,122 +0,0 @@
-<?php
-
-namespace Chompy\Jobs;
-
-use Chompy\Stat;
-use Carbon\Carbon;
-use Chompy\Services\Rogue;
-use Illuminate\Bus\Queueable;
-use Chompy\Events\LogProgress;
-use Chompy\Traits\ImportToRogue;
-use Illuminate\Queue\SerializesModels;
-use Illuminate\Queue\InteractsWithQueue;
-use Illuminate\Contracts\Queue\ShouldQueue;
-use Illuminate\Foundation\Bus\Dispatchable;
-
-class ImportFacebookSharePosts implements ShouldQueue
-{
-    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels, ImportToRogue;
-
-    /**
-     * The path to the stored csv.
-     *
-     * @var string
-     */
-    protected $filepath;
-
-    /**
-     * Stat counter.
-     */
-    protected $stats;
-
-    /**
-     * Total record counter.
-     */
-    protected $totalRecords;
-
-    /**
-     * Create a new job instance.
-     *
-     * @return void
-     */
-    public function __construct($filepath)
-    {
-        $this->filepath = $filepath;
-
-        $this->stats = $this->statsInit();
```

Comment:
Seems this is the only job that was posting anything to Stathat (and that calls the `statsInit ` function defined over in the [`ImportToRogue` trait](https://github.com/DoSomething/chompy/blob/0569eb68780d47d2e4e49b7cdcb46e5356599614/app/Traits/ImportToRogue.php#L16)